src/IDPDiscoveryController.js

```diff
@@ -44,7 +44,11 @@ function (Okta, PrimaryAuthController, PrimaryAuthModel, IDPDiscoveryForm, IDPDi
       // If social auth is configured, 'socialAuthPositionTop' will determine
       // the order in which the social auth and primary auth are shown on the screen.
       if (options.settings.get('hasConfiguredButtons')) {
-        this.add(CustomButtons, {prepend: options.settings.get('socialAuthPositionTop')});
+        // CustomButtons needs current controller as parameter
```

Comment:
nit: I'd suggest moving this comment inside of the `options` object, since it only pertains to the `afterError` event.

Something like:

```js
this.add(CustomButtons, {
  options: {
    // To trigger an afterError event, we require the current controller
    currentController: this 
  }
});
```